.cloudbuild/e2e.yaml

```diff
@@ -1,7 +1,8 @@
 # yaml-language-server: $schema=https://json.schemastore.org/cloudbuild.json
 substitutions:
   _NODE: node:18-alpine
-  _CLOUD_SDK_VERSION: 413.0.0
+  # Any version from 416.0.0 should suffice. See: https://github.com/GoogleCloudPlatform/gsutil/issues/1663#issuecomment-1411161340
```

Comment:
Let's keep it pinned but I don't think we need to keep the comment in the code after this fix.

```suggestion
```